book.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'item'
+
+# Class representing a Book
+class Book < Item
+  attr_accessor :publisher, :cover_state, :label
+
+  def initialize(params = {})
+    super(params)
+    @publisher = params[:publisher]
+    @cover_state = params[:cover_state]
+    @label = params[:label]
+  end
```

Comment:
[Optional] For 3 parameters, it's good to use no-object in the middle but do this : 
def initialize(publisher, cover_state, label) 
when you define